Firestore/core/src/remote/serializer.cc

```diff
@@ -1198,31 +1168,24 @@ Filter Serializer::DecodeUnaryFilter(
               "Unexpected UnaryFilter.which_operand_type: %s",
               unary.which_operand_type);
 
-  StatusOr<FieldPath> field =
-      FieldPath::FromServerFormat(DecodeString(unary.field.field_path));
-
-  if (!field.ok()) {
-    reader->set_status(field.status());
-    return InvalidFilter();
-  }
+  auto field = DecodeFieldPath(reader, unary.field.field_path);
```

Comment:
Similar nit: similar statements above use an explicit types, please make consistent (I'm fine with either way).

Reply:
I just want to show off my versatile coding skills here. Fixed.